apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/AbstractClaimRdaSink.java

```diff
@@ -405,6 +405,8 @@ private void mergeBatch(long maxSeq, Collection<RdaChange<TClaim>> changes) {
                 entityManager.merge(change.getClaim());
                 insertCount += getInsertCount(change.getClaim());
               } else {
+                // We would expect this to have been filtered by the RdaSource so it is safe
+                // to stop processing with an exception here.
                 throw new IllegalArgumentException(
                     "RDA API DELETE changes are not currently supported");
```

Comment:
Would this still be needed if we're skipping them before this?

Reply:
I left it in place for safety.